esp-wifi/examples/esp_now.rs

```diff
@@ -0,0 +1,100 @@
+#![no_std]
```

Comment:
I guess this whole file shouldn't be here

Reply:
Oops! Dodgy rebase :sweat_smile: